packages/paste-website/src/pages/components/chat-composer/index.mdx

```diff
@@ -0,0 +1,225 @@
+    ChatComposer,
+    AutoScrollPlugin,
+    Button,
+    SendIcon,
+  }}
+>
+  {ChatDialogExample}
+</LivePreview>
+
+## Guidelines
+
+### About Chat Composer
+
+A Chat Composer is an input made for users to type rich chat messages. Chat Composer can be used as one part of larger chat user interfaces to provide a seamless authoring experience.
+
+### Accessibility
+
+Chat Composer supports a variety of aria attributes which are passed into the content editable region of the component.
+
+### Lexical and Plugins
+
+Chat Composer is built ontop of [Lexical](https://lexical.dev/). The editor itself is extensible and follows a declarative approach to configuration via JSX. Developers can leverage a
+wide variety of existing plugins or write their own to augment the behavior of Chat Composer. To include new Chat Composer behavior provide plugins via the `children` prop.
+
+## Examples
+
+### Basic
+
+<LivePreview scope={{ChatComposer}} language="jsx">
+  {`<ChatComposer config={{namespace: 'foo', onError: (e) => { throw e } }} ariaLabel="A Basic Chat Composer" />`}
+</LivePreview>
+
+### With Placeholder
+
+<LivePreview scope={{ChatComposer}} language="jsx">
+  {`<ChatComposer config={{namespace: 'foo', onError: (e) => { throw e } }} placeholder="Type Here..." ariaLabel="A Basic Chat Composer" />`}
+</LivePreview>
+
+### With Initial Value
+
+<LivePreview scope={{ChatComposer}} language="jsx">
+  {`<ChatComposer config={{namespace: 'foo', onError: (e) => { throw e } }} initialValue="This is my initial value" placeholder="Type Here..." ariaLabel="A Basic Chat Composer" />`}
+</LivePreview>
+
+### With Max Height
+
+<LivePreview scope={{ChatComposer}} language="jsx">
+  {`<ChatComposer config={{namespace: 'foo', onError: (e) => { throw e } }} maxHeight="size10" ariaLabel="A Basic Chat Composer" />`}
+</LivePreview>
+
+## Usage Guide
+
+### API
+
+#### Installation
+
+```bash
+yarn add @twilio-paste/chat-composer - or - yarn add @twilio-paste/core
+```
+
+#### Usage
+
+```jsx
+import {ChatComposer} from '@twilio-paste/core/chat-composer';
+
+export const BasicChatComposer = () => (
+  <ChatComposer
+    config={{
+      namespace: 'foo',
+      onError: (e) => {
+        throw e;
+      },
+    }}
+    ariaLabel="A basic chat composer"
+    placeholder="Type here"
+  />
+);
+```
+
+#### Props
+
+| Prop            | Type                                                        | Description                                                                               | Default                                   |
+| --------------- | ----------------------------------------------------------- | ----------------------------------------------------------------------------------------- | ----------------------------------------- | --- |
+| `config`        | `LexicalComposerProps['initialConfig']`                     | Configuration provided to the Lexical Composer                                            |                                           |
```

Comment:
question: do we want to specify that these are undefined by default or is leaving that field in the table blank enough.